pages/members/[slug].js

```diff
@@ -177,12 +198,43 @@ const MemberPage = ({ member, loadError }) => {
                 </Linkify>
               </p>
             }
+            </div>
+
+      <div className="flex flex-col items-start md:w-2/3">
+        <div className="page-title flex justify-between">
+          <h3 className="mt-6 mb-4">Upcoming events {member.screenname} is going to:</h3>
+          {/* <div className="action">
+            { member && member.roles.includes('event-creator') &&
+              <Link href="/events/create">
+                <a className="btn-primary">Create event</a>
+              </Link>
+            }
+          </div> */}
+        </div>
+        
+          <UpcomingEvents
+          allowCreate
+          limit={ 30 }
+          page={ 1 }
+          labelLink={null}
+        />
+          <div className="flex flex-col items-start mb-10">
```

Comment:
This needs to be implemented - I already shared the API specs in your previous MR.